benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -854,7 +854,21 @@ void GraphicsBenchmarkApp::Render()
     }
     submitInfo.pFence = frame.renderCompleteFence;
 
+    Timer timerSubmit;
+    timerSubmit.Start();
     PPX_CHECKED_CALL(GetGraphicsQueue()->Submit(&submitInfo));
+    double t = timerSubmit.MillisSinceStart();
+
+    uint64_t frameCount     = GetFrameCount();
+    mSubmissionTime.average = (mSubmissionTime.average * frameCount) / (frameCount + 1) + t / (frameCount + 1);
```

Comment:
Should we use the existing metric framework for this: https://github.com/google/bigwheels/blob/main/src/ppx/application.cpp#L1747 ? 
That way we can also export this metric

Reply:
I feel like there's other metrics in this benchmark that would be good to use the metrics framework for. Maybe we should just do some refactoring in a separate PR after this to address them?

Reply:
yeah, separate PR would be better.

Reply:
sounds good